src/App.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useMemo, useState } from "react";
+import "./App.css";
+import { BrowserRouter, Route, Routes } from "react-router-dom";
+import Homepage from "./Components/HomePage";
+import About from "./Components/About";
+import Navbar from "./Components/Navbar";
+import Create from "./Components/Create";
+import TravelDetails from "./Components/TravelDetails";
+import Login from "./Components/Login";
```

Comment:
Pilu, esetleg beszeljetek roviden arrol, hogy ilyesmit index.ts fileba is ki lehet tenni.
Persze egy ilyen projectnel ez mar overengineering, nem azt mondom, hogy feltelneul szukseges.